src/test/java/seedu/address/model/plan/PlanTest.java

```diff
@@ -46,6 +46,33 @@ public void equals() {
         assertFalse(BALI.equals(editedBaliPlan));
     }
 
+    @Test
+    public void isSamePlan() {
```

Comment:
I love how you added comments for every assert statement! Makes it clear and understandable.